sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -108,6 +110,12 @@ _rom_ext_interrupt_vector:
    */
   .option push
   .option norelax
+  /**
+   * Enable position independent code, since this routine must run in any address
+   * until the address translaton get configured.   
+   */
+  .option pic
+  .option norvc
```

Comment:
Out of curiosity is the `norvc` required? That indicates to me that we are relying on an instruction not being optimized which we should avoid.

Reply:
You are right, I added it initially because it`s easier to debug and decode 32 bits instructions.
I'll remove it.